app/models/user.rb

```diff
@@ -19,4 +19,11 @@ class User < ApplicationRecord
   def downcase_email
     email.downcase!
   end
+
+  class << self
+    def User.digest(string)
```

Comment:
có self thì bỏ User đi e
```suggestion
    def digest string
```